docs/design/ovs-pipeline.md

```diff
@@ -406,42 +406,54 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### AntreaPolicyEgressRuleTables (45, 49)
+### AntreaPolicyEgressRuleTable (45)
 
-For these two tables, you will need to keep in mind the ACNP
+For this table, you will need to keep in mind the ACNP
 [specification](#antrea-networkpolicy-crd-implementation)
 that we are using.
 
-These two tables are used to implement the egress rules across all Antrea-native policies.
-Depending on the tier to which the ACNP or ANP belongs to, the rules will be 
-installed in a table corresponding to that tier. The egress table to tier mappings 
-is as follows:
-
+This table is used to implement the egress rules across all Antrea-native policies,
+except for policies that are created in the Baseline Tier. Antrea-native policies
+created in the Baseline Tier will be enforced after K8s NetworkPolicies, and their
+egress rules are installed in the [EgressDefaultTable] and [EgressRuleTable]
+respectively, i.e.
 ```text
```

Comment:
add empty line before code block for proper website rendering

Reply:
Done.